charts/velero/templates/deployment.yaml

```diff
@@ -110,16 +107,16 @@ spec:
           {{- if .Values.credentials.useSecret }}
             {{- if eq $provider "aws" }}
             - name: AWS_SHARED_CREDENTIALS_FILE
-              value: /credentials/cloud
+              value: /credentials/{{ .Values.credentials.existingSecretKey }}
```

Comment:
I'm not sure I agree with this change, as it fundamentally changes how Velero works from `velero install`, resulting in branching installation paths that we maintainers have to support.

I like it in theory, as it provides support for multiple cloud providers, but starts to introduce a big drift between the Helm chart and the `velero install` command.

Reply:
That's a pretty easy swap. We can pull the secret at that key and mount it at `/credentials/cloud` if you like. That said, as you note, multiple providers is a bit of an open question at the moment. I asked about it above in the plugin change, though perhaps the answer is simply it's multiple provider "ready".

How about we drop it at `/credentials/cloud` so it's the same as `velero install` and then if you change velero install to support multiple providers there we can update the chart to place it the same